app/controllers/api/v1/base_controller.rb

```diff
@@ -27,7 +27,7 @@ def log_exception exception
   end
 
   def authenticate_user_using_x_auth_token
-    user_email = params[:id].presence
+    user_email = params[:email].presence || request.headers['X-Auth-Email']
```

Comment:
@vipulnsward We are picking up the `auth_token` only from the headers so If an API call can be made with one header it can also be made with another header(for the email). What am I missing?